phenotrex/ml/clf/svm.py

```diff
@@ -57,6 +63,23 @@ def __init__(self, C: float = 5., penalty: str = "l2", tol: float = 1.,
             ("clf", classifier)
         ])
 
+    def train(self, records: List[TrainingRecord], train_explainer: bool = True, *args, **kwargs):
+        # must override train method here to append shapexplainer training afterwards.
+        # This is not required for XGBoost as XGboost trains a shap model internally per default.
+        super().train(records=records, *args, **kwargs)
+        clf = self.pipeline.named_steps['clf']
+        if train_explainer:
+            # must use k-means to summarize, else intractable at inference time with KernelExplainer
+            self.logger.info('Training SHAP KernelExplainer.')
+            self.logger.info(f'Running KMeans with k={KMEANS_N_CLUSTERS} on background data...')
+            data = shap.kmeans(self._get_raw_features(records).toarray(), k=KMEANS_N_CLUSTERS)
+            self.shap_explainer = shap.KernelExplainer(
+                clf.predict_proba,
+                data,
+                link="logit"
```

Comment:
```suggestion
                link="logit",
```
If we already use this black-like style, adding trailing commas could shorten future diffs slightly (when adding additional argument after `link`)